.travis.yml

```diff
@@ -34,6 +34,8 @@ env:
   matrix:
     # Keep the TESTS sorted by name and grouped by type
     - TESTS="smoke/test_certauthority_root"
+
+    - TESTS="smoke/test_diagnostics"
```

Comment:
Please keep the name sorted, include your file under an existing test section/block, don't add a new job/runner.

Reply:
Included test to the section with routers and ssvm tests.

Reply:
No, you should keep it sorted by name as the file suggests.

Reply:
- TESTS="smoke/test_accounts
             smoke/test_affinity_groups
             smoke/test_affinity_groups_projects
             smoke/test_deploy_vgpu_enabled_vm
             smoke/test_deploy_vm_iso
             smoke/test_deploy_vm_root_resize
             smoke/test_deploy_vm_with_userdata
             smoke/test_deploy_vms_with_varied_deploymentplanners
             **smoke/test_diagnostics**
             smoke/test_disk_offerings
             smoke/test_dynamicroles
             smoke/test_global_settings
             smoke/test_guest_vlan_range"